### ISSUE TITLE
Bug: Keeps asking to grant accessibility permission even when it already has it

### ISSUE THREAD

**Issue by AdityaAnand1** [lwouis/alt-tab-macos#52]
On macOS catalina 👇 

![image](https://user-images.githubusercontent.com/13274079/67502429-7edc9c00-f6a3-11e9-8427-62c7e0d3605d.png)

**Comment by AdityaAnand1** [comment]
Ok, I'm not sure what I did, but it just worked. I unchecked once and then launched app again and then checked the permission again and then it worked.

Perhaps it has something to do with me previously having this installed and then removing it and then adding it again?

@lwouis Please close if this doesn't seem like it could be an issue.

**Comment by lwouis** [comment]
The permissions UX in macOS is pretty bad. When I iterate on the app, I often have to go in these preferences, and quickly `uncheck` > `check` the permission. I imagine that being the scene, the OS tracks the app bundle as a unique app, and if you install a new version, even if the name is the same, and the unique bundle ID is the same, it will consider it as a new app.

This is reasonable on Apple's part. What is not reasonable I think, is that their UI shows only 1 line and 1 checkbox, so the user doesn't know the currently checked app is actually the previous version. You see "AltTab.app" and you think "oh it's already checked so why isn't it working?" but it's actually v1 that was checked before, and current v2 is not granted permission. unchecking and rechecking the same box will actually grant permissions to v2.

Basically your ticket is bad UX from macOS :/
